pkg/process/config/yaml_config.go

```diff
@@ -303,8 +303,12 @@ func (a *AgentConfig) initProcessDiscoveryCheck() {
 
 	// We don't need to check if the key exists since we already bound it to a default in InitConfig.
 	// We use a minimum of 10 minutes for this value.
-	a.CheckIntervals[DiscoveryCheckName] =
-		time.Duration(math.Max(float64(config.Datadog.GetDuration(key(root, "interval"))), float64(10*time.Minute)))
+	discoveryInterval := config.Datadog.GetDuration(key(root, "interval"))
+	if discoveryInterval < discoveryMinInterval {
+		discoveryInterval = discoveryMinInterval
+		_ = log.Warnf("Invalid interval for process discovery (<= %sm) using default value of %[1]s", discoveryMinInterval.String())
```

Comment:
nit
```suggestion
		_ = log.Warnf("Invalid interval for process discovery (<= %s) using default value of %[1]s", discoveryMinInterval.String())
```

Reply:
Oh! I didn't notice that I forgot to fix that. Thanks so much :)